### DISCUSSION TITLE
Javascript integration help...

### DISCUSSION THREAD

**Discussion by mcconnon12** [swellstores/community#467] [Help] [answered]
Hey Everyone,

I am integrating Partnero affiliate software with Swell's hosted cart checkout. I did a test run and it works! Which is great but I need to get the dynamic data to pass through...

There is sign up and sales data that I need to dynamically pass through.

How can I alter the code to do that?

Thanks in advance,
Shawn

See images from Partnero...
![Screenshot 2024-01-05 114105](https://github.com/swellstores/community/assets/119345161/c632a475-5b99-4a4b-a41f-1ad3181025fa)
![Screenshot 2024-01-05 114044](https://github.com/swellstores/community/assets/119345161/4ef7aebf-2804-49dc-a287-4d2fc0d7ec16)

**Comment by sags95** [comment, marked as answer]
Hi @mcconnon12, does Partnero also offer an API for this functionality? It might be more reliable to handle this through events with webhooks (rather than client-side). 

Related to this, we also have a new feature we're calling Swell Apps which allows for new ways to build integrations on Swell. I think there would be a lot of useful functionality for you to try out if you're interested. It's currently in Alpha, but if you sent an email to apps-alpha@swell.is I can get you access.

**Reply by mcconnon12** [reply]
They do have an API. I'll send an email now. Thanks.
